### PR TITLE
githubstudent's jamaican jerk chicken

### PR DESCRIPTION
**Pull request by githubstudent**
11055405_844274338971011_1260567503_n

## TODO

- [x] Template of recipe
- [x] Ingredients
- [x] Directions
- [x] Attributing original author with Images

### CODE REVIEW

**Comment by githubstudent** [comment]
@githubteacher: can you review this? it's ready to go!

File: _posts/2015-03-24-jamaican-jerk-chicken.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "Jamaican Jerk Chicken"
+date:   2015-03-24 10:30
+categories: main
+---
+
+# Jamaican Jerk Chicken
+
+![githubstudent-jamaican-jerk-chicken](https://igcdn-photos-d-a.akamaihd.net/hphotos-ak-xaf1/t51.2885-15/11055405_844274338971011_1260567503_n.jpg)
```
Comment (githubteacher) [inline _posts/2015-03-24-jamaican-jerk-chicken.md]: This image should actually be the image you added into the Opening Post of this pull request. This isn't a clear and easy url from instagram. I'd recommend against using this.

Comment (githubstudent) [inline _posts/2015-03-24-jamaican-jerk-chicken.md]: This has been updated in a recent commit

**Comment by githubteacher** [comment]
This looks good now!

**Pull request merged**